templates/partials/question.html

```diff
@@ -44,7 +50,6 @@
       "id": "question-guidance-" ~ question.id,
       "classes": "ons-u-mb-m"
     }) %}
-      {% set contents = question.guidance.contents %}
```

Comment:
How come this changed?

Reply:
I removed it because it is a duplicate of line 42